src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -358,7 +362,6 @@ function ReportPreview({
     const shouldShowPendingSubtitle = numberOfPendingRequests === 1 && numberOfRequests === 1;
 
     const isPayAtEndExpense = ReportUtils.isPayAtEndExpenseReport(iouReportID, allTransactions);
-    const isArchivedReport = ReportUtils.isArchivedRoomWithID(iouReportID);
     const [archiveReason] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${iouReportID}`, {selector: ReportUtils.getArchiveReason});
 
     const getPendingMessageProps: () => PendingMessageProps = () => {
```

Comment:
Hey @rlinoz, do you know of the top of your head what `type` of report will have `ARCHIVE_REASON.BOOKING_END_DATE_HAS_PASSED`? As in `chat`, `expense` or `iou`? We're refactoring some of the archived code so might have to make some changes based on this.

Reply:
Travel reports that are related to a booking that you had to pay in person, in this case we create a placeholder report that gets archived when the end date of the booking has passed

https://github.com/Expensify/Web-Expensify/blob/bb1305f554dad2a193e9bd1c7891a213399fe6dd/script/bwm/ArchiveReportAtBookingEndDate.php#L23

Reply:
Ok cool, it makes sense, thank you! After these changes, since it is an expense report, it will be able to be commented on. But there will still be no actions like `Submit`, `Approve`, `Pay` on these reports. Does that sound ok? cc @trjExpensify

Reply:
There aren't any action buttons in this case. It's just a placeholder booking confirmation, which we intended to then archive using the pattern applied to chat reports when the booking date has passed. This is the visual ref:

<img width="294" alt="image" src="https://github.com/user-attachments/assets/85d73cba-5468-4102-8812-27f13a6697c7">

Reply:
Ah okay, so even though this is an expense report, do we still want to continue archiving it using the chat pattern? Or do we want to switch to the expense report archive pattern where the composer is visible?

The `This booking is archived now that the trip date has passed...` header text will still be visible with either approach that we take.

Reply:
Using the archive pattern still makes sense to me, because our intention is to communicate it as archived as displayed above. So if it doesn't follow that same pattern as how other archived chats show up in the product, it's confusing.